test/org/sagebionetworks/bridge/models/AppleAppSiteAssociationTest.java

```diff
@@ -0,0 +1,41 @@
+package org.sagebionetworks.bridge.models;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+import org.sagebionetworks.bridge.models.studies.AppleAppLink;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.google.common.collect.Lists;
+
+public class AppleAppSiteAssociationTest {
+
+    @Test
+    public void createsCorrectJson() throws Exception {
+        AppleAppLink detail1 = new AppleAppLink("appId1", Lists.newArrayList("/appId1/","/appId1/*"));
+        AppleAppLink detail2 = new AppleAppLink("appId2", Lists.newArrayList("/appId2/","/appId2/*"));
+        
+        AppleAppSiteAssociation assoc = new AppleAppSiteAssociation(Lists.newArrayList(detail1, detail2));
+        
+        JsonNode node = new ObjectMapper().valueToTree(assoc);
```

Comment:
Similarly, do we ever need to deserialize this?

Reply:
No again. We never receive these as input.